x-pack/plugins/security_solution/public/resolver/view/panels/cube_for_process.tsx

```diff
@@ -43,7 +43,7 @@ export const CubeForProcess = memo(function ({
       className={className}
       width="2.15em"
       height="2.15em"
-      viewBox="0 0 100% 100%"
+      viewBox="0 0 34 34"
```

Comment:
:information_source: Thanks, had this on my list from the other big PR last week.